R/tribble.R

```diff
@@ -9,6 +9,9 @@
 #'
 #' @param ... Arguments specifying the structure of a `tibble`.
 #'   Variable names should be formulas, and may only appear before the data.
+#'   If a function needs to be applied to the data for a column (e.g. `factor`),
+#'   this should apear after the `~`, e.g. factor(column_A). Other arguments
+#'   cab be provided as needed.
```

Comment:
Regarding the documentation, I'll expand that. I've also now removed the possibility of giving the column name as a character string rather than symbol when using a conversion function, since it could potentially be confusing.

The other option one could consider for this functionality is syntax along the lines of:

    colA ~ lubridate::ymd(.)

However, it moves the column name to the LHS of the ~, which is different to unconverted columns.